python/private/stage2_bootstrap_template.py

```diff
@@ -0,0 +1,501 @@
+# This is a "stage 2" bootstrap. We can assume we've running under the desired
+# interpreter, with some of the basic interpreter options/envvars set.
+# However, more setup is required to make the app's real main file runnable.
```

Comment:
I assume that most of the contents got copied from the main template? Maybe we should mark it as such?

The `CamelCase` usage in functions is something that is legacy, I presume.

Reply:
I tried a few things, but I'm not sure they took. The github UI never showed it as a copy. I'm not sure what the commit history showed, but since it all gets squashed anyways, I'm not sure it would matter.